galaxy.yml

```diff
@@ -1,11 +1,10 @@
 namespace: "ibm"
 name: "cloud"
-version: "0.0.1-beta1"
+version: "1.0.0"
 readme: "README.md"
 authors:
-    - "Mujeebur Rahman"
     - "Harini Kantareddy"
-    - "Kavya Handadi"
```

Comment:
@umarali-nagoor , Please revert the changes of the authors. Add you name along with the exiting authors.

Same applies to the authors in modules.